build-tools/packages/build-cli/src/BasePackageCommand.ts

```diff
@@ -129,21 +129,20 @@ export abstract class PackageCommand<
 		}
 
 		try {
-			await async.mapLimit(
-				packages,
-				this.flags.concurrency,
-				async (details: PackageWithKind) => {
-					started += 1;
+			await async.mapLimit(packages, this.flags.concurrency, async (pkg: PackageWithKind) => {
+				started += 1;
+				updateStatus();
+				try {
+					await this.processPackage(pkg, pkg.kind);
+					succeeded += 1;
+				} catch (error) {
```

Comment:
implicitly typing error as `any` is more unsafe than needed.


```suggestion
				} catch (error: unknown) {
```

is our preferred pattern.

I'd expect is to object to what you have here due to implicit any: do you not have noImplicit any enabled in this package?

Reply:
Looks like that option is not enabled in our shared tsconfig: https://github.com/microsoft/FluidFramework/blob/c600fec19bd4f3d600293bb32d832d18218523d5/common/build/build-common/ts-common-config.json#L13

That seems... wrong. But also out of scope here. I'll set it to true for this project and see how many changes it requires.

Reply:
I enabled noImplicitAny for the project, but it didn't appear to have any effect.

Reply:
> I enabled noImplicitAny for the project, but it didn't appear to have any effect.

Scratch that. It caused a build failure because of missing prettier types, which I've added. I wonder how hard it would be to enable this more broadly?